crates/bevy_render/src/view/window.rs

```diff
@@ -215,8 +229,28 @@ pub fn prepare_windows(
                 CompositeAlphaMode::PostMultiplied => wgpu::CompositeAlphaMode::PostMultiplied,
                 CompositeAlphaMode::Inherit => wgpu::CompositeAlphaMode::Inherit,
             },
+            // TODO: Use an sRGB view format here on platforms that don't support sRGB surfaces. (afaik only WebGPU)
+            view_formats: vec![],
         };
 
+        // This is an ugly hack to work around drivers that don't support MSAA.
```

Comment:
this is indeed pretty ugly, but at least `ResMut<Msaa>` doesn't conflict with anything here so it shouldn't affect perf. good incentive to get #7194 done :)